qiskit_addon_sqd/fermion.py

```diff
@@ -66,6 +79,316 @@ def load(cls, filename):
         with np.load(filename) as data:
             return cls(data["amplitudes"], data["ci_strs_a"], data["ci_strs_b"])
 
+    def rdm(self, rank: int = 1, spin_summed: bool = False) -> np.ndarray:
+        """Compute reduced density matrix."""
+        # Reason for type: ignore: mypy can't tell the return type of the
+        # PySCF functions
+        sci_vector = _as_SCIvector(self.amplitudes, (self.ci_strs_a, self.ci_strs_b))
+        if rank == 1:
+            if spin_summed:
+                return make_rdm1(sci_vector, self.norb, self.nelec)  # type: ignore
+            return make_rdm1s(sci_vector, self.norb, self.nelec)  # type: ignore
+        if rank == 2:
+            if spin_summed:
+                return make_rdm2(sci_vector, self.norb, self.nelec)  # type: ignore
+            return make_rdm2s(sci_vector, self.norb, self.nelec)  # type: ignore
+        raise NotImplementedError(
+            f"Computing the rank {rank} reduced density matrix is currently not supported."
+        )
+
+    def orbital_occupancies(self) -> tuple[np.ndarray, np.ndarray]:
```

Comment:
Same comment as above regarding prefixing this function name with a verb

Reply:
Same response as above.